frontend/src/stores/levels/actions.ts

```diff
@@ -0,0 +1,47 @@
+import { get, post, put, deleteRequest } from '@/services/http'
+import { levels } from './state'
+import type { Level } from '@/domain/Level'
+
+// Function to sort levels based on minCoefficient and maxCoefficient
+function sortLevels(levels: Level[]): Level[] {
+  return levels.slice().sort((a, b) => {
+    return a.minCoefficient - b.minCoefficient || a.maxCoefficient - b.maxCoefficient
+  })
+}
+
+async function fetchLevels(): Promise<void> {
```

Comment:
This function is useless, you'll fetch the levels of just one level, not all levels